test/js_test.go

```diff
@@ -2343,15 +2351,21 @@ func TestJetStreamImportDirectOnly(t *testing.T) {
 		}
 	}
 
-	// Cannot subscribe with JS context from another account right now.
-	if _, err := js.SubscribeSync("ORDERS"); err != nats.ErrJetStreamNotEnabled {
-		t.Fatalf("Expected an error of '%v', got '%v'", nats.ErrJetStreamNotEnabled, err)
+	// Cannot subscribe with JS context from another account unless the stream is specified
+	if _, err := js.SubscribeSync("ORDERS"); err != nats.ErrBoundJetStreamStream {
+		t.Fatalf("Expected an error of '%v', got '%v'", nats.ErrBoundJetStreamStream, err)
 	}
-	if _, err = js.SubscribeSync("ORDERS", nats.BindStream("ORDERS")); err != nats.ErrJetStreamNotEnabled {
-		t.Fatalf("Expected an error of '%v', got '%v'", nats.ErrJetStreamNotEnabled, err)
+	if _, err = js.SubscribeSync("ORDERS", nats.BindStream("ORDERS")); err != nats.ErrBoundJetStreamDurable {
+		t.Fatalf("Expected an error of '%v', got '%v'", nats.ErrBoundJetStreamDurable, err)
 	}
-	if _, err = js.PullSubscribe("ORDERS", "d1", nats.BindStream("ORDERS")); err != nats.ErrJetStreamNotEnabled {
-		t.Fatalf("Expected an error of '%v', got '%v'", nats.ErrJetStreamNotEnabled, err)
+	if sub, err = js.PullSubscribe("ORDERS", "d1", nats.BindStream("ORDERS")); err != nil {
+		t.Fatalf("Expected no error, got '%v'", err)
+	} else if m, err := sub.Fetch(1); err != nil {
+		t.Fatalf("expected no error, got %v", err)
+	} else if len(m) != 1 {
+		t.Fatalf("expected one message, got %d", len(m))
+	} else if err = m[0].Ack(); err != nil {
```

Comment:
Can you change to `AckSync` I think it would work as well

```suggestion
	} else if err = m[0].AckSync(); err != nil {
```